docs/api/search-autocomplete-service-resource.md

```diff
@@ -50,6 +50,7 @@ skip        | URL    | integer | no       | The number of results to skip, for p
 take        | URL    | integer | no       | The number of results to return, for pagination
 prerelease  | URL    | boolean | no       | `true` or `false` determining whether to include [pre-release packages](../create-packages/prerelease-packages.md)
 semVerLevel | URL    | string  | no       | A SemVer 1.0.0 version string 
+packageType | URL    | string  | no       | The package type to use to filter packages
```

Comment:
We should mention this was added in autocomplete version 3.5.0. Also, the Versioning section should mention that autocomplete version 3.5.0 exists